src/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-left: 2px;
+}
+
+.stars > div {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 > div {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-top: 2px;
+}
+
+.stars--1 > div:nth-child(1) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--1 > div:nth-child(n + 2) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 > div:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 > div:nth-child(n + 3) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
Use class names and check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)